docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md

```diff
@@ -7,99 +7,100 @@ description: >
 
-### Inputs:
-{{< imgproc nodered_inputs Fit "800x800" >}}{{< /imgproc >}}
+### 1st example: Connecting external sensor technology
 
-**With the help of the inputs you can tap different data sources. Like for example:**
-- OPC/UA ([documentation for this node](https://flows.nodered.org/node/node-red-contrib-opcua))
-- Siemens S7 ([documentation for this node](https://flows.nodered.org/node/node-red-contrib-s7))
-- TCP/IP ([documentation for this node](https://flows.nodered.org/flow/bed6f676d088670d7e1bc298943338b5))
-- Rest API  ([documentation for this node](https://cookbook.nodered.org/http/create-an-http-endpoint))
-- Modbus  ([documentation for this node](https://flows.nodered.org/node/node-red-contrib-modbus))
-- MQTT ([documentation for this node](https://cookbook.nodered.org/mqtt/))
+You would like to determine the output and machine condition of a cutting machine.
+
+Used Sensors:
 
-**Interaction with sensorconnect (Plug and Play connection of IO-Link Senosors):**
+- Light Barrier
+- Button Bar
 
-With the help of Sensorconnect, different sensors can be connected quickly and easily via an IFM gateway. The sensor values are automatically extracted from the software stack and made available via [MQTT](http://www.steves-internet-guide.com/mqtt-works/).
+With the help of Sensorconnect, sensors can be connected quickly and easily via an IFM gateway. The sensor values are automatically extracted from the software stack and made available via [MQTT](https://docs.umh.app/docs/concepts/mqtt/). 
 
-To get a quick and easy overview of the available MQTT messages and topics we recommend the [MQTT Explorer](http://mqtt-explorer.com/). If you don't want to install any extra software you can use the MQTT-In node to subscribe to all available topics by subscribing to  `#` and then direct the messages of the MQTT in nodes into a debugging node. You can then display the messages in the nodered debugging window and get information about the topic and available data points.
+*1. Connect Sensors*
 
+TODO: Picture or illustration of how to connect sensors
 
-Topic structure: `ia/raw/<transmitterID>/<gatewaySerialNumber>/<portNumber>/<IOLinkSensorID>`
+*2. Make the connected sensors visible to our system*
 
-#### Example for ia/raw/
+Based on an IP address, which is assigned to each sensor (or gateway?), the sensor can be integrated into our system. Only then is it possible to read out sensor values. The adaptation of the IP range is required.
 
-Topic: `ia/raw/2020-0102/0000005898845/X01/210-156`
+TODO: ADAPTION OF IP-RANGE
 
-This means that the transmitter with the serial number `2020-0102` has one ifm gateway connected to it with the serial number `0000005898845`. This gateway has the sensor `210-156` connected to the first port `X01`.
+*3. Creating a flow*
 
-```json
-{
-"timestamp_ms": 1588879689394, 
-"distance": 16
-}
-```
+In Node-RED basically three pieces of information must be communicated to the system.
```

Comment:
In order to contextualise the resulting data points with the help of the United Manufacturing Hub, 3 identifiers are necessary: